BrowserKit/Sources/Common/Theming/ThemeType.swift

```diff
@@ -24,15 +27,8 @@ public enum ThemeType: String {
             return .default
         case .dark:
             return .black
-        }
-    }
-
-    public func getThemedImageName(name: String) -> String {
-        switch self {
-        case .light:
-            return name
-        case .dark:
-            return "\(name)_dark"
```

Comment:
This is currently not being used and confirmed with design that it is not needed for private mode as it will use dark mode assets.